tecken/api/views.py

```diff
@@ -628,3 +631,69 @@ def nones_to_zero(obj):
 
     context = {"stats": numbers}
     return http.JsonResponse(context)
+
+
+# Store a result for 10 minutes
+SYMINFO_RESULT_TIMEOUT = 600
+
+# Indicates there's nothing in the cache
+NO_VALUE_IN_CACHE = object()
+
+
+@metrics.timer_decorator("lookup_by_syminfo.timing")
```

Comment:
We should adjust `timer_decorator` to call transform function `on_return` or `on_error` so they can adjust the timing metric with tags. I'll toss an item in markus for that.